src/main/java/edu/ksu/canvas/attendance/form/CourseConfigurationValidator.java

```diff
@@ -34,8 +34,8 @@ public void validate(Object target, Errors errors) {
                 return;
             }
 
-            if ((courseConfigurationForm.getPresentPoints() > 100 || courseConfigurationForm.getPresentPoints() < 0) || (courseConfigurationForm.getTardyPoints() > 100 || courseConfigurationForm.getTardyPoints() < 0)
-                    || (courseConfigurationForm.getExcusedPoints() > 100 || courseConfigurationForm.getExcusedPoints() < 0) || (courseConfigurationForm.getAbsentPoints() > 100 || courseConfigurationForm.getAbsentPoints() < 0)) {
+            if ((Double.parseDouble(courseConfigurationForm.getPresentPoints()) > 100 || Double.parseDouble(courseConfigurationForm.getPresentPoints()) < 0) || (Double.parseDouble(courseConfigurationForm.getTardyPoints()) > 100 || Double.parseDouble(courseConfigurationForm.getTardyPoints()) < 0)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce the number of conditional operators (7) used in the expression (maximum allowed 3). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1067)